### DISCUSSION TITLE
What's the key to yank to system clipboard?

### DISCUSSION THREAD

**Discussion by richardhttps** [helix-editor/helix#4421] [Q&A] [answered]
~~I'd like to be able to copy and paste text from and to system clipboard, and between multiple instances. I couldn't find a key in docs.
There are yank commands, but from the mapping section in the docs it says mapping to commands is not yet supported.~~

Edit: Apprently I misread the docs. Sorry for the confusion.

**Comment by David-Else** [comment, marked as answer]
space and y/Y

**Reply by richardhttps** [reply]
Thanks! I overlooked the space mode
